src/components/NumberOptions/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box } from '../Box';
+import Steps, { Step } from '../Steps';
+import { PValue } from './styled';
+
+interface Props {
+  options?: number[];
+  isPercentage?: boolean;
+  onChange: (value: number) => void;
+  currentValue: number;
+  isDisabled?: boolean;
+  variant: 'step' | 'plain' | 'box';
+}
+
+const NumberOptions = ({
+  options = [25, 50, 75, 100],
```

Comment:
remove Percent component from interface and use NumberOptions everywhere but make sure its working with new component because there is little difference in Percent and NumberOptions component

Reply:
For me to remove do I need this component to be in this repo?

Reply:
NumberOptions is taken from Percent component. so ultimately we should be able to remove Percent component from interface because NumberOptions is almost same as Percent

Reply:
I created this PR for this https://github.com/pangolindex/interface/pull/447